index.html

```diff
@@ -34,65 +34,215 @@ <h3 class="masthead-brand">resPOSTa!</h3>
 				<div class="inner cover">
 					<h1 class="cover-heading">Scared to Ask Questions in Class?</h1>
 					<p class="lead">This totally anonymous and real-time updating question moderated system permits students to ask more questions in a typical class setting.
-						Simply join a question room using a simple name (such as comp3111).</p>
-						<p class="lead" class="col-md-2 col-md-offset-5">
-							<form name="form" ng-submit="join()" class="navbar-form navbar-left" role="search">
-								<input type="text" name = "input" class="form-control large"
-								ng-model="example.text"
-								ng-pattern="example.word" required
-								ng-trim="false"
-								autofocus
-								aria-describedby="sizing-addon1"
-								placeholder="Room name">
-								<a href="#" class="btn btn-lg btn-default"
-								ng-hide="form.input.$error.required || form.input.$error.pattern"
-								ng-click="join()"
-								>Join</a>
-							</form>
-						</p>
+						Simply join a question room using a simple name (such as comp3111).
+                    </p>
+                    <p class="lead" class="col-md-2 col-md-offset-5">
+                        <form name="form" ng-submit="join()" class="navbar-form navbar-left" role="search">
+                            <input type="text" name = "input" class="form-control large"
+                            ng-model="roomName.text"
+                            ng-pattern="roomName.word" required
+                            ng-trim="false"
+                            autofocus
+                            aria-describedby="sizing-addon1"
+                            placeholder="Room name">
+                            <a href="#" class="btn btn-lg btn-default"
+                            ng-hide="form.input.$error.required || form.input.$error.pattern"
+                            ng-click="join()"
+                            >Join</a>
+                        </form>
+                    </p>
+                    <div class="alert alert-warning" role="alert" ng-show="form.input.$error.required">
```

Comment:
Before entering the first letter, this alert is already displayed. 
You should couple the display of this alert to the pressing of the 'Create a Room' button (6 lines later on line 60).
Create an issue for this. 

![bildschirmfoto 2015-11-19 um 22 01 57](https://cloud.githubusercontent.com/assets/1174752/11272625/3476b796-8f09-11e5-8662-439da9ed3ae1.png)